data_models.rb

```diff
@@ -0,0 +1,21 @@
+require 'data_mapper'
+
+if ENV['RACK_ENV'] != 'production'
+  require 'dotenv'
+  Dotenv.load('.env')
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'])
+
+class User
+  include DataMapper::Resource
+
+  property :id,         Serial
+  property :email,      String, :required => true, :format => :email_address
+  property :nick_name,  String, :required => true, :unique => true
+  property :password,   String, :required => true, :length => 6..20
+end
```

Comment:
This looks like a good start. Eventually you'll want to encrypt passwords and store the password hash (not plain text).

Also, be aware that this schema design for `User` allows for multiple users to have the same email address, which could cause some confusion down the line. Probably better to add a uniqueness constraint to your email field.